packages/bot-web-ui/src/components/audio/audio.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 
-const Audio = () => (
+const Audio: React.FC = () => (
     <>
-        <audio id='announcement' src={`${__webpack_public_path__}media/announcement.mp3`} autostart='false' />
-        <audio id='earned-money' src={`${__webpack_public_path__}media/coins.mp3`} autostart='false' />
-        <audio id='job-done' src={`${__webpack_public_path__}media/job-done.mp3`} autostart='false' />
-        <audio id='error' src={`${__webpack_public_path__}media/out-of-bounds.mp3`} autostart='false' />
-        <audio id='severe-error' src={`${__webpack_public_path__}media/i-am-being-serious.mp3`} autostart='false' />
```

Comment:
Why are we removing autostart prop?

Reply:
Audio HTML tag has no attribute `autostart`. it has autoPlay but as this component works fine without using autoPlay I have not added it

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio#attributes